fe/fe-core/src/main/java/org/apache/doris/qe/SessionVariable.java

```diff
@@ -978,6 +984,9 @@ public void setMaxJoinNumBushyTree(int maxJoinNumBushyTree) {
     @VariableMgr.VarAttr(name = ENABLE_FOLD_NONDETERMINISTIC_FN)
     public boolean enableFoldNondeterministicFn = true;
 
+    @VariableMgr.VarAttr(name = MINIDUMP_PATH)
+    public String minidumpPath = "default";
```

Comment:
nereids_minidump_path, btw use empty string to present default maybe better

Reply:
get